gilbreth_perception/src/segmentation_node.cpp

```diff
@@ -111,18 +177,16 @@ void cloudCb(const sensor_msgs::PointCloud2ConstPtr &cloud_msg)
   ec.setInputCloud(scene);
   ec.extract(cluster_indices);
 
-  if(cluster_indices.empty())
-  {
+  if (cluster_indices.empty()) {
     ROS_ERROR("Segmentation found no clusters after applying Clustering Extraction");
     return;
   }
 
   // Extracting and publishing cluster
   pcl::PointIndices::Ptr indices(new pcl::PointIndices);
   pcl::ExtractIndices<pcl::PointXYZ> extract;
-
-  for (int i = 0; i < cluster_indices.size(); i++)
-  {
+  int voxel_grid_size = 32;
```

Comment:
Should this be made into a global constant variable?

Reply:
Modified.